packages/server/src/api/muster/muster.controller.ts

```diff
@@ -218,6 +219,146 @@ class MusterController {
 
     res.json(closestMuster ? buildMusterWindow(unit, closestStart, closestEnd, closestMuster) : null);
   }
+
+  /**
+   * This method returns the normalized rate (0 - 1.0) of compliance on a given
+   * date for a given unit, against  of the unit's muster configs.
+   * @param ReportDateQuery - query param for the date used to check compliance
+   * @param unitName - API param to specify the unit to check for compliance
+   */
+  async getMusterComplianceByDate(req: ApiRequest<{unitName:string}, null, DateQuery>, res: Response) {
```

Comment:
Parameter req is not described in JSDoc

Reply:
Yeah this is a little unorthodox, I wanted to communicate the API params here. Maybe I just get rid of them.

Reply:
params removed